CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](http://semver.org).
 ## Supported Release [4.3.0]
 ### Summary
 
+This release is a rollup of new features and fixes.
+
+#### Added
+- Install `apt-transport-https` if using Debian 8, 9 or Ubuntu 14.04, 16.04.
+- Adds a boolean option `direct` to proxy settings to bypass `https_proxy` if not set.
+- Adds facter facts for `dist-upgrade` apt updates.
+
+#### Changed
+- Update class is now private.
+- (MODULES-4265) Detect security updates from multiple sources.
```

Comment:
This can include a hyperlink to the ticket.

Reply:
Added as hyperlink. Moved to 'Fixed' section as it looks more like a bug fix.

Reply:
I don't think you want the stars in front of the hyperlink.

Reply:
Removed the asterisks, was copy and pasting the format from an earlier build note but it didn't really work.